sklearn/metrics/classification.py

```diff
@@ -603,12 +603,19 @@ def f1_score(y_true, y_pred, labels=None, pos_label=1, average='binary',
     The F1 score can be interpreted as a weighted average of the precision and
     recall, where an F1 score reaches its best value at 1 and worst score at 0.
     The relative contribution of precision and recall to the F1 score are
-    equal. The formula for the F1 score is::
+    equal. The formula for the F1 score in the simple ``binary`` case is::
 
         F1 = 2 * (precision * recall) / (precision + recall)
 
-    In the multi-class and multi-label case, this is the weighted average of
-    the F1 score of each class.
+    In the multi-class and multi-label case, the overall F1 score can be
+    calculated as a weighted average of the F1 scores for the classes/labels.
+    Depending on the averaging methods, there are different overall F1 scores::
+    ``macro-averaging`` gives the arithmetic mean of the F1 scores; ``micro-
+    averaging`` uses the total true positives, false negatives and false
```

Comment:
Agree for the incorrect part. I was careless. 'Weighted' could be both 'macro' and 'weighted' options for the 'average' parameters. But I took it out to avoid confusion.